src/DevTools/Extension/components/Shell/Shell.tsx

```diff
@@ -17,58 +18,97 @@ export const Shell = () => {
   const [selectedShellTab, setSelectedShellTab] = useSelectedShellTab();
```

Comment:
Should we add prop to set default panel position? like you did for the trigger buttont?

Maybe something like?  `panelPosition?: "top" | "right" | "bottom" | "left"` 🤔